public/app/features/datasources/components/picker/DataSourceModal.tsx

```diff
@@ -24,17 +26,54 @@ interface DataSourceModalProps {
   recentlyUsed?: string[];
   enableFileUpload?: boolean;
   fileUploadOptions?: DropzoneOptions;
+  reportedInteractionFrom?: string;
 }
 
+const MAP_BUILT_IN_DS_TO_EVENT_ITEM: Record<string, string> = {
+  grafana: 'use_mock_data_ds',
```

Comment:
Agree.

Maybe we can just report the even with something like:
`reportInteraction('event', {dsType: ds.meta.type}));`

This will give us information about the most used types, but also if the built-in types are more used now. WDYT @thanos-karachalios ?